src/fparser/two/Fortran2003.py

```diff
@@ -2482,26 +2488,53 @@ def tostr(self):
         return '(%s, %s)' % tuple(self.items)
 
 
-class Ac_Implied_Do_Control(Base):  # R471
-    """
-    <ac-implied-do-control> = <ac-do-variable> = <scalar-int-expr> ,
+class Ac_Implied_Do_Control(Base):
+    '''
+    Fortran2003 rule R471
+    Specifies the syntax for the control of an implicit loop within an
+    array constructor.
+
+    <ac-implied-do-control> is <ac-do-variable> = <scalar-int-expr> ,
```

Comment:
Could you drop the < and > please to conform to the "new" style (that more closely follows the specification formatting

Reply:
Sure, done.